### DISCUSSION TITLE
Unexpected download and upload behavior

### DISCUSSION THREAD

**Discussion by crogonint** [felddy/foundryvtt-docker#261] [Q&A]
Look here.. WHY am I downloading the FoundryVTT package at 42KBPS, and UPLOADING at the same speed?!

What is this nonsense? Are you uploading my signed package somewhere?

**Comment by felddy** [comment]
I am unable to diagnose your personal connectivity issues.  AWS does have a support article that may be of use:
https://aws.amazon.com/premiumsupport/knowledge-center/s3-troubleshoot-slow-downloads-uploads/

The container entrypoint and support tools only connect to:
- [foundryvtt.com](https://foundryvtt.com) for license retrieval if `FOUNDRY_USERNAME` and `FOUNDRY_PASSWORD` are set.
- AWS S3 bucket to download a FoundryVTT tarball using the presigned URL provided either by [foundryvtt.com](https://foundryvtt.com) or the passed in via the `FOUNDRY_RELEASE_URL` environment variable.

The container does not upload.

Please feel free to review the [entrypoint source code and tools](https://github.com/felddy/foundryvtt-docker/tree/develop/src) to verify these claims.

**Comment by crogonint** [comment]
The container DID upload. I don't have time to troubleshoot your nonsense. I uninstalled the entire docker pile of nonsense and recovered to a previous date.

I don't know why you got the impression I was complaining about speeds. The problem is that at the same time I was DOWNLOADING, it was mirroring and uploading to somewhere else. ONLY THIS APPLICATION DOES THIS.

..or rather did. I don't have time to sort out who's hacking what, so as I said, I uninstalled the entire thing and recovered Linux to a previous date.
